app/src/autocomplete.js

```diff
@@ -128,10 +138,13 @@ class Autocomplete {
     return Math.floor(inputWidth / 35);
   }
 
-  _source(params, locale) {
+  _source(params, locale, clickAnalyticsEnabled) {
     return (query, callback) => {
-      this.index.search({...params, query, optionalWords: getOptionalWords(query, locale)})
-        .then((content) => { callback(this._reorderedHits(content.hits)); });
+      this.index.search({...params, clickAnalytics: clickAnalyticsEnabled, query, optionalWords: getOptionalWords(query, locale)})
+        .then((content) => { 
+          const hits = this._addPositionToHits(content.hits, content.queryID);
```

Comment:
I assume you only want to call this function if `clickAnalyticsEnabled` is true, right?

Reply:
ah yes, indeed!